docs/src/main/asciidoc/_admin-guide/subsystem-configuration/Logging.adoc

```diff
@@ -9,11 +9,19 @@ ifdef::env-github[]
 :warning-caption: :warning:
 endif::[]
 
-The overall server logging configuration is represented by the logging
+[abstract]
+--
+The overall server logging configuration is represented by the `logging`
 subsystem. It consists of four notable parts: `handler` configurations,
 `logger` and the `root logger` declarations (aka log categories) and
 logging profiles. Each logger references a handler (or set of
-handlers). Each handler declares the log format and output:
+handlers). Each handler declares the log format and output.
+--
+
+[TIP]
+The WildFly Model Reference provides a complete description of the link:./wildscribe/subsystem/logging/index.html[`logging` subsystem].
```

Comment:
Minor, but should we consider opening this in a new window?

Reply:
Perhaps "`logging` subsystem's management API"?

Reply:
I changed the link text.

@jamezp I'm not keen on changing the target of links. If the user wants to open it in a new window/tab, their web browser let them do that.
I find more annoying to have multiple browsers windows that open tabs :)